tests/generators/kzg_4844/main.py

```diff
@@ -399,6 +399,20 @@ def case05_verify_blob_kzg_proof():
             'output': None
         }
 
+    # Edge case: Incorrect `G1_POINT_AT_INFINITY` Proof
+    blob = BLOB_RANDOM_VALID1
+    commitment = spec.blob_to_kzg_commitment(blob)
+    proof = spec.G1_POINT_AT_INFINITY
+    assert not spec.verify_blob_kzg_proof(blob, commitment, proof)
+    yield 'verify_blob_kzg_proof_case_incorrect_proof_point_at_infinity', {
```

Comment:
Minor nit. I think we should group this with the other "incorrect" tests. So move this test case below the `verify_blob_kzg_proof_case_incorrect_proof` test case.